.github/ISSUE_TEMPLATE/developer-onboarding.md

```diff
@@ -37,8 +38,8 @@ cf login -a api.fr.cloud.gov  --sso
 ## Documents to Review
 
 - [ ] [Team Charter](https://docs.google.com/document/d/1xhMKlW8bMcxyF7ipsOYxw1SQYVi-lWPkcDHSUS6miNg/edit), in particular our Github Policy
-- [ ] [Architecture Decision Records](docs/architecture/decisions)
-- [ ] [Github Policy](CONTRIBUTING.md)
+- [ ] [Architecture Decision Records](https://github.com/cisagov/dotgov/tree/main/docs/architecture/decisions)
+- [ ] [Github Policy](https://github.com/cisagov/dotgov/tree/main/CONTRIBUTING.md)
```

Comment:
Nonblocking thought: wondering if Contribution Policy might be clearer, especially since the document is called "Contributing"? In looking at the Markdown preview I also initially thought this was a link to Github's policy